sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java

```diff
@@ -139,7 +139,7 @@ public void processElement(ProcessContext c) throws Exception {
         tableReference.setProjectId(
             c.getPipelineOptions().as(BigQueryOptions.class).getProject());
         tableDestination = new TableDestination(
-            tableReference, tableDestination.getTableDescription());
+            tableReference, tableDestination.getTableDescription(),tableDestination.getTimePartitioning());
```

Comment:
Checkstyle complains usually if you don't let spaces after commas. Fixing this would make CI happy and it will pass.